### PR TITLE
Add recipe for stompest

### PR DESCRIPTION
**Pull request by fcollonval**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/stompest/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+test:
+  imports:
+    - stompest
+    - stompest._backwards
+    - stompest.config
+    - stompest.error
+    - stompest.protocol
+    - stompest.sync
+    - stompest.sync.examples
+    - stompest.tests
```
Comment (xhochy) [inline recipes/stompest/meta.yaml:34]:
```suggestion
```

File: recipes/stompest/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  host:
+    - pip
+    - python
+  run:
+    - python
+
+test:
+  imports:
+    - stompest
+    - stompest._backwards
```
Comment (xhochy) [inline recipes/stompest/meta.yaml:28]: Only test public modules.
```suggestion
```

File: recipes/stompest/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+{% set version = "2.3.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  - url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+    sha256: 83a4d9d9627028953e7bec71a958a60488243f7ea6f41c2a428ad95f533cedce
+  - path: .
```
Comment (xhochy) [inline recipes/stompest/meta.yaml:11]:
```suggestion
```

Comment (fcollonval) [reply recipes/stompest/meta.yaml:11]: Thanks for reviewing. The License file is not part of the pip package. I use the `path: .` trick to include one. What is the good way to include it?

**Pull request merged**